.goreleaser.yml

```diff
@@ -37,8 +37,6 @@ builds:
   - 6
   - 7
   ignore:
-    - goos: darwin
-      goarch: arm
```

Comment:
I think we still need to ignore this - `darwin/arm` is 32-bit ARM, and it was dropped in Go 1.15, whereas what we want is `darwin/arm64` (which wasn't ignored before, but it used to refer to the iOS port).

Reply:
Ah, okay thanks. I wonder if we need to do anything special to get `arm64` to build or if it'll just work now that we build on 1.16. 🤔

Reply:
I think it should _just work_ https://github.com/goreleaser/goreleaser/pull/1956